automation/src/main/java/org/wso2/testgrid/automation/parser/TestNgResultsParser.java

```diff
@@ -120,10 +121,16 @@ public TestNgResultsParser(TestScenario testScenario, String testLocation) {
      */
     @Override
     public void parseResults() throws ResultParserException {
-        final Path dataBucket = DataBucketsHelper.getOutputLocation(testScenario.getTestPlan());
+        Path dataBucket = DataBucketsHelper.getOutputLocation(testScenario.getTestPlan());
+        dataBucket = Paths.get(dataBucket.toString(), "test-outputs/scenarios", testScenario.getOutputDir(),
+                testScenario.getName());
+        logger.info(testScenario.getName());
         Set<Path> inputFiles = getResultInputFiles(dataBucket);
 
-        final Path outputLocation = DataBucketsHelper.getOutputLocation(testScenario.getTestPlan());
+        Path outputLocation = DataBucketsHelper.getOutputLocation(testScenario.getTestPlan());
+        outputLocation = Paths.get(outputLocation.toString(), "test-outputs/scenarios",
```

Comment:
`test-outputs/scenarios` can be replaced with `TestGridConstants.TEST_RESULTS_DIR`